bench/composer.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "automapper/bench",
```

Comment:
I would prefer to split that in another PR please 🙏 (And maybe add a quick note about the output in documentation too ?)

Reply:
This is mainly for internal, to check some behavior and keeps them versioned, so we can link to them to why we do that in pull request or issue, not sure if we want that in doc ? could add a readme.md though

Reply:
Let's add a readme then 👍

Reply:
done